pkg/build/android.go

```diff
@@ -132,6 +240,48 @@ func (a android) build(params Params) (ImageDetails, error) {
 	return details, nil
 }
 
+func (a android) makeCmd(params Params, extraArgs ...string) (*exec.Cmd, error) {
+	target := targets.Get(targets.Linux, params.TargetArch)
+	args := LinuxMakeArgs(target, params.Compiler, params.Linker, params.Ccache, "")
+	args = append(args, extraArgs...)
+	cmd := osutil.Command("make", args...)
+	if err := osutil.Sandbox(cmd, true, true); err != nil {
+		return cmd, err
+	}
+	return cmd, nil
+}
+
+func (a android) runCmd(cmd *exec.Cmd, kernelDir string) error {
+	// Pre-append prebuilts to path.
+	absPrebuiltsPath := filepath.Join(kernelDir, prebuiltsPath)
+	env := os.Environ()
+	for idx, envVar := range env {
+		if strings.HasPrefix(envVar, "PATH=") {
+			curPath := os.Getenv("PATH")
+			env[idx] = fmt.Sprintf("PATH=%v:%v", absPrebuiltsPath, curPath)
```

Comment:
Do you only need to use the prebuilts path when there's at least one already specified PATH variable? Or do you want it to be always set?

Also, note that if there are several same keys, only [the last one will be used](https://pkg.go.dev/os/exec#Cmd). So we don't really have to adjust each one.

Reply:
Ah that makes sense, thanks! I adjusted it to always be set.

Reply:
Oh sorry! I just realized I hadn't pushed the change. It should be correct now.